client/scss/components/_dropdown.legacy.scss

```diff
@@ -111,7 +111,7 @@
     .button,
     button {
       border-radius: 0;
-      font-size: 0.95em;
+      font-size: 1.06em;
```

Comment:
I think we could just remove this `font-size` definition! No point in redefining this if there’s already a definition that matches what we’re after.

```suggestion
```